app/templates/ember-community-survey-2018.hbs

```diff
@@ -17,19 +17,10 @@
   @sectionTitle="Ember Version"
 >
   <:charts>
-    <div>
-      <Highcharts
-        @chartOptions={{this.newReleaseAdoption.options}}
-        @data={{this.newReleaseAdoption.data}}
-      />
-    </div>
-
-    <div class="mt-3">
-      <Highcharts
-        @chartOptions={{this.overallAdoption.options}}
-        @data={{this.overallAdoption.data}}
-      />
```

Comment:
I removed the second chart because the new spline chart (`newReleaseAdoption`) displays the same data as the bar chart (`overallAdoption`), but is easier to read and contains one additional year of data.